Support/lib/executable.rb

```diff
@@ -0,0 +1,93 @@
+require 'shellwords'
+
+module Executable
+  class NotFound < RuntimeError; end
+
+  class << self
+    # Try to find the Ruby executable `name`. Return an array containing the
+    # command(s) needed to run the executable. If the executable could not be
+    # found, `Executable::NotFound` is raised with a message describing the
+    # problem.
+    #
+    # Typical usage looks like this:
+    #
+    #     Dir.chdir(ENV['TM_PROJECT_DIRECTORY']) if ENV['TM_PROJECT_DIRECTORY']
+    #     begin
+    #       executable = Executable.find('rspec')
+    #       system(*executable, '--some', '--additional', 'args')
+    #     rescue Executable::NotFound => e
+    #       # Executable not found, so use fallback / display alert / …
+    #       # `e.message` contains detailled error message.
```

Comment:
Thx, fixed in 0ec1709! (That word is an old foe of mine, because in German it is spelled “detailliert” with two L :)